### ISSUE TITLE
Crash when attempting to extract embedded objects from SV6

### ISSUE THREAD

**Issue by RollingStar** [IntelOrca/OpenRCT2#1030]
Downloading this park, copying the two WAVEs into /Data/ and the one object into /ObjData/, and trying to load the park as a saved game causes a crash on Windows 8.1.

http://www.nedesigns.com/park/3326/

http://www.nedesigns.com/park/3326/h2h7-r1-cedar-points-raptor/download/

For what it's worth. regular RCT2 crashes after extracting (like it does with many embedded parks) but can load it the second time.

**Comment by duncanspumpkin** [comment]
This one is crashing because of heavy abuse of 8cars. I can get it running if I disable some extra checks we added to the station logic but obviously that's not a proper fix. The issue is caused by the part ride sections that are used as scenery. If you are determined to load it make the changes i mention in #777. 

The reason you have to copy that objData file is because the file has the incorrect checksum. I tried a couple tests a couple nights ago and it all packs correctly if you use the correct checksum.

Basically the same issue as #777.

**Comment by RollingStar** [comment]
Dupe of #777.
